src/mono/browser/browser.proj

```diff
@@ -340,7 +341,8 @@
       { "identity": "WasmNativeStrip",              "defaultValueInRuntimePack": "$(WasmNativeStrip)" },
       { "identity": "WasmSingleFileBundle",         "defaultValueInRuntimePack": "$(WasmSingleFileBundle)" },
       { "identity": "WasmEnableSIMD",               "defaultValueInRuntimePack": "$(WasmEnableSIMD)" },
-      { "identity": "WasmEnableExceptionHandling",  "defaultValueInRuntimePack": "$(WasmEnableExceptionHandling)" }
+      { "identity": "WasmEnableExceptionHandling",  "defaultValueInRuntimePack": "$(WasmEnableExceptionHandling)" },
```

Comment:
It can be done in a follow-up if we want to get this in. The element is named `BoolPropertiesThatTriggerRelinking` and this is not a bool property. Let's rename it to general `PropertiesThatTriggerRelinking` and the other side as well, please